platform/import-export/src/actions/ImportAsJSONAction.js

```diff
@@ -130,8 +136,23 @@ define(['zepto'], function ($) {
      * @private
      */
     ImportAsJSONAction.prototype.rewriteId = function (oldID, newID, tree) {
-        tree = JSON.stringify(tree).replace(new RegExp(oldID, 'g'), newID);
-        return JSON.parse(tree);
+        let newIdKeyString = this.openmct.objects.makeKeyString(newID);
+        let oldIdKeyString = this.openmct.objects.makeKeyString(oldID);
```

Comment:
Nitpick: I understand naming `newID` and `oldID` variables are not part of your changes, but it would be good to follow the same naming pattern as the rest of the file, i.e. `newId` and `oldId` here as well.

Reply:
Done!